.teamcity/Gradle_Util/buildTypes/WarmupEc2Agent.kt

```diff
@@ -0,0 +1,24 @@
+package Gradle_Util.buildTypes
+
+import common.gradleWrapper
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import jetbrains.buildServer.configs.kotlin.v2019_2.Requirement
+import jetbrains.buildServer.configs.kotlin.v2019_2.RequirementType
+
+object WarmupEc2Agent : BuildType({
+    uuid = "980bce31-2a3e-4563-9717-7c03e184f4a4"
```

Comment:
Nit: I don't remember using this format for UUIDs. IIRC the uuid is the project id prefixed. Could you please look it up.

Reply:
I think the format of the uuid doesn't really matter, as long as it doesn't change over time.